lib/joken.ex

```diff
@@ -129,10 +129,12 @@ defmodule Joken do
   @spec peek_header(bearer_token) :: {:ok, claims} | {:error, error_reason}
   def peek_header(token) when is_binary(token) do
     with {:ok, %{"protected" => protected}} <- expand(token),
-         {:ok, decoded_str} <- Base.url_decode64(protected, padding: false),
+         {:decode64, {:ok, decoded_str}} <-
```

Comment:
The same thing happens on `peek_claims`. Can we update there too in one go?

Thanks :)

Reply:
Copy-paste made me test this wrong! Fixing it now :)